examples/hello_world/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "deno.enable": true
+}
```

Comment:
This shouldn't be committed to the project. Editor configuration files are personal.

Reply:
This is actually very standard for deno projects to check into git.

Reply:
Because everyone is doing it wrong, doesn't mean it's standard 😶

Reply:
Removing the `/vscode/settings.json` file requires one more step of the vscode-using-tutorial-follower to get their vscode, configured to use the builtin typescript formatter, to correctly format/highlight code.

How likely is it that an alternative (non deno) typescript compiler is introduced to this hello world example in the future?